src/coreclr/src/tools/r2rdump/R2RDump.cs

```diff
@@ -154,7 +154,7 @@ public IEnumerable<ReadyToRunSection> NormalizedSections()
 
         public IEnumerable<ReadyToRunMethod> NormalizedMethods()
         {
-            IEnumerable<ReadyToRunMethod> methods = _r2r.Methods;
+            IEnumerable<ReadyToRunMethod> methods = _r2r.Methods.Values.SelectMany(sectionMethods => sectionMethods);
```

Comment:
Similar to one of my previous comments, why cannot we assign the _r2r.Methods.Values to methods?
There are two similar cases below too.

Reply:
Please let me know if I'm missing something but I believe we need to use SelectMany to flatten the 2-level list to 1 level as I mentioned above by stating that, after my refactoring to support per-assembly (per-R2R section) module lists the Values is actually of type <code>IEnumerable&lt;List&lt;ReadyToRunMethod&gt;&gt;</code> and foreach is apparently unable to iterate such multi-level lists by default.